packages/types/index.d.ts

```diff
@@ -82,4 +91,13 @@ export interface FilecoinSnapApi {
   getBalance(): Promise<string>;
   exportPrivateKey(): Promise<string>;
   configure(configuration: SnapConfig): Promise<void>;
+}
+
+export interface Transaction {
+  hash: string;
```

Comment:
@mpetrunic I am not sure what props should describe tx, so I defined few generic ones for now.